module/bdev/raid/bdev_raid.c

```diff
@@ -2349,15 +2858,9 @@ static void
 raid_bdev_event_base_bdev(enum spdk_bdev_event_type type, struct spdk_bdev *bdev,
 			  void *event_ctx)
 {
-	int rc;
-
 	switch (type) {
 	case SPDK_BDEV_EVENT_REMOVE:
-		rc = raid_bdev_remove_base_bdev(bdev, NULL, NULL);
-		if (rc != 0) {
-			SPDK_ERRLOG("Failed to remove base bdev %s: %s\n",
-				    spdk_bdev_get_name(bdev), spdk_strerror(-rc));
-		}
+		raid_bdev_remove_event_base_bdev(bdev);
```

Comment:
The function doesn't have return value.
Should we handle the error `raid_bdev_find_base_info_by_bdev(bdev);` fails?

Reply:
This error shouldn't never happen. However, if it comes, we can't do so much because something bad happened in bdev layer.